src/controllers/image-request.ts

```diff
@@ -111,13 +91,33 @@ export const handle = async (
     };
   }
 
+  let imagePath = request.uri;
```

Comment:
Keen to avoid using `let` as much as possible with a preference of `const`

Happy for the logic for determining image path to be extracted into a utility function